issuetracker/src/main/java/org/openjdk/skara/issuetracker/jira/JiraIssue.java

```diff
@@ -278,6 +284,7 @@ public void removeLabel(String label) {
 
     @Override
     public void setLabels(List<String> labels) {
+        this.labels = null;
```

Comment:
Both `GitHubPullRequest` and `GitHubMergeRequest` update the internal `labels` field in this method instead of making it null.

Reply:
I think we update the internal labels in GitHubPullRequest and GitHubMergeRequest because we could get the return value of the update request. However, for Jira Issue, I read the document of Jira REST API, it shows that it wouldn't have return value of the update request, it will just return Status 204.
Reference:
https://developer.atlassian.com/server/jira/platform/jira-rest-api-examples/

Reply:
So we update the internal labels in GitHubPullRequest and GitLabMerge request could help us reduce one request. But update the internal labels here, I think we wouldn't get any benefit.

Reply:
NVM, my bad, update here would also reduce one request, I will change it